python/pdstools/adm/ADMDatamart.py

```diff
@@ -286,10 +286,20 @@ def import_data(
                 df2 = df2.join(df1.select(pl.col("ModelID").unique()), on="ModelID")
 
         if self.import_strategy == "eager":
-            if df1 is not None:
-                df1 = df1.collect().lazy()
-            if df2 is not None:
-                df2 = df2.collect().lazy()
+            try:
+                if df1 is not None:
+                    df1 = df1.collect().lazy()
+                if df2 is not None:
+                    df2 = df2.collect().lazy()
+            except pl.ComputeError as e:
```

Comment:
I don't think this'd be the right way of doing it. This is just a catch-all exception catcher (ComputeError is very generic) and it doesn't actually _have_ to mean it's something to do with the timestamps. How to catch that though, I'm not sure.

Also, I wouldn't raise a general Exception but I'd try to raise a more specific error. Datetime itself raises a ValueError, so that's already better.